ci.nix

```diff
@@ -29,7 +29,7 @@
       ghc884 = true;
       ghc8101 = false;
       ghc8102 = true;
-      ghc8102-experimental = true;
+      ghc81020201118 = true;
```

Comment:
tbh, maybe we should just set this to `false`, and assume that it's adequately covered by the `8.10.2` one